pythran/optimizations/constant_folding.py

```diff
@@ -7,7 +7,6 @@
 from pythran.analyses.ast_matcher import DamnTooLongPattern
 
 import ast
-import numpy
```

Comment:
I think numpy is required here. It is not in the code but eval may need to load it. (If I correctly remember... sorry for not documenting this ...)